Source/WebCore/platform/graphics/freetype/FontPlatformDataFreeType.cpp

```diff
@@ -322,4 +322,40 @@ HbUniquePtr<hb_font_t> FontPlatformData::createOpenTypeMathHarfBuzzFont() const
 }
 #endif
 
+#if PLATFORM(WPE)
```

Comment:
Why is this WPE specific?

Reply:
Oh, this is a mistake. Since this file is freetype specific this conditional check is not necessary for sure.